experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/timeline.html

```diff
@@ -0,0 +1,25 @@
+<div class="container-fluid py-4">
+  <!-- Timeline Row -->
+  <div class="d-flex align-items-center justify-content-between position-relative">
+    <!-- Draft Status -->
+    <div class="border border-dark rounded p-3 text-center flex-grow-1 mx-2 {% if experiment.status == 'Draft' %}bg-primary text-white{% else %}bg-light{% endif %}">
+      <div>Draft</div>
```

Comment:
I'd suggest making these something like `<h5>` so they stand out from the dates a bit.

Reply:
sure sure

Reply:
<img width="1589" alt="Screenshot 2024-09-23 at 8 04 55 PM" src="https://github.com/user-attachments/assets/13c5d87f-4f0b-4e20-9418-f3f4970bea83">